app/src/main/java/com/greenaddress/abcore/MainActivity.java

```diff
@@ -177,6 +183,9 @@ public boolean onCreateOptionsMenu(final Menu menu) {
     public boolean onOptionsItemSelected(final MenuItem item) {
         // Handle item selection
         switch (item.getItemId()) {
+            case R.id.dashboard:
```

Comment:
I’m thinking that for consistency with other Android apps, the dashboard (“home”) of the app shouldn’t be on the menu, as we can get to it by using the “back” button? If so, this should be removed from all instances of the menu. But do correct me if I’m wrong, I’m not an Android user so maybe my expectation is wrong here.

Reply:
I am not an android user either :-). (except for ABCore)

But what you are saying makes sense to me and it did cross my mind as I was coding it. I will make this change and update the PR. 

Speaking of which, my long term plan was to propose having the the top menu only have 'Settings' and 'About' (from which the user would have to hit the back button) and Dashboard, Peers, Console, Log be part of a bottom menu. I will post this proposal in the Telegram group so other can weigh in as well. But here is a rough mockup of how things would looks like.

![image](https://user-images.githubusercontent.com/48868830/60761455-b264bf00-9ffd-11e9-8d26-5e6a1ede4e0a.png)

Reply:
Adding a new commit to remove the menu from the non-Main activities. Screenshots now look like:
![image](https://user-images.githubusercontent.com/48868830/60763000-f9fc4280-a01f-11e9-9700-2f5be486fb25.png)

Reply:
Oh, sorry, actually I didn't mean to suggest removing the menu from the other activities, but to remove the "dashboard" option from where the menu appears, because you'd get back to it by hitting "back".

That said, I think that removing the menu from the other activities works as well :)

In any case, the "dashboard" entry in the menu should still be removed

Reply:
Regarding the "bottom bar menu" idea: I can see you're an iOS person :) I think these are much less popular on Android than on iOS, probably makes more sense to do a top bar (https://material.io/design/components/tabs.html)

Reply:
>That said, I think that removing the menu from the other activities works as well :)
>In any case, the "dashboard" entry in the menu should still be removed

Did this one in the latest commit. 

>probably makes more sense to do a top bar 

Makes sense :-)
Will change it to top bar/tab when I send the PR for that.